### DISCUSSION TITLE
Add manual type of  graph layout

### DISCUSSION THREAD

**Discussion by Hann1bal** [reaviz/reagraph#116] [Ideas]
What if add custom layout, with manual set x,y coords for edges and links? I mean if i'm have a extra large graph and i adapted algorithm for server side, and calculate radial tree-graph, and client( react ) only draw it. 
Something like that:
![image](https://github.com/reaviz/reagraph/assets/20579904/92a248f4-3fb7-449e-aa12-594016cd1ba4)

**Comment by popovitsj** [comment]
This is now possible, actually. See https://reagraph.dev/?path=/story/demos-layouts--custom

**Reply by amcdnl** [reply]
Yup!!! Would love to see what other layouts people use!
